### PR TITLE
Add example for port range to win_firewall_rule_module

### PR DESCRIPTION
**Pull request by cossss**
##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
Sample for opening range of ports.

People often ask me if this is supported because it's not in the docs

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME

- win_firewall_rule_module

### CODE REVIEW

File: lib/ansible/modules/windows/win_firewall_rule.py [hunk 1]
```diff
@@ -144,4 +144,14 @@
     protocol: tcp
     state: present
     enabled: yes
+
```
Comment (acozine) [inline lib/ansible/modules/windows/win_firewall_rule.py]:
```suggestion

```

File: lib/ansible/modules/windows/win_firewall_rule.py [hunk 2]
```diff
@@ -144,4 +144,14 @@
...
+    
+- name: Firewall rule to allow port range
+  win_firewall_rule:
+    name: Sample port range
+    localport: 5000-5010
+    action: allow
+    direction: in
+    protocol: tcp
+    state: present
+    enabled: yes
```
Comment (acozine) [inline lib/ansible/modules/windows/win_firewall_rule.py]:
```suggestion
    enabled: yes
```

**Review by acozine — COMMENTED** [review]
Thanks @cossss for adding this example. The suggestions below should fix the whitespace issues and get this PR passing our tests.

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @cossss for the Ansible documentation fix!

**Pull request merged**